### DISCUSSION TITLE
Filter out-of-date projects from search

### DISCUSSION THREAD

**Discussion by callumbwhyte** [umbraco/Umbraco.Packages#10] [Ideas] [answered]
When searching for a package on Our a lot of results are returned, many of which aren't too relevant.

It seems the version drop down next to the search box has no effect, and it is unclear what order the results are being shown in.

Additionally, results often contain older (likely deprecated) packages from the v4 days... (are these still relevant?)

See for example: https://our.umbraco.com/search?q=robots&cat=project

Besides fixing the filtering, perhaps projects that haven't been updated in a while (arbitrary time, 2 years?) could be automatically flagged as "deprecated" and therefore hidden from search by default? Authors can be notified, and can easily override this in the package settings

**Comment by jmayntzhusen** [comment]
A couple of things in one issue here, thanks for mentioning them though 🙂 

In the future when we do a project on Our improvements we will revisit this 👍

**Comment by dawoe** [comment]
Also filtering out packages on last update date is a no go in my opinion. I have a package that hasn't been updated in 1.5 year but still works on all V7 sites from 7.3 and up.

Maybe it would be better to display a warning that it has not been updated in a while and it may have issues on the latest versions is a better option

**Comment by jmayntzhusen** [comment]
Yes several ways to tackle this, I think a revisit on the whole voting whether things work on a current version would be better. 

Testing packages with new releases before they are released would be really nice, and something we need to discuss a proper format and procedure for at some point 🙂

**Comment by LottePitcher** [comment]
Perhaps a 'last activity' concept so that if the package has been confirmed as working on releases then this counts as still 'active'.  Would be great to improve the 'Package Compatibility' lists as lots of them are showing as untested which is somewhat off-putting. So yes, a review of the voting procedure would be good to do!

**Comment by ronaldbarendse** [comment, marked as answer]
The Marketplace site only shows v8 and higher packages, plus you can now filter on a specific CMS version. The Our packages section will be retired at some time, especially now v7 will be EOL on 30 September.

I'm not sure how we'll handle EOL versions on the Marketplace, but a sensible thing to do is to only show packages that support any currently supported CMS version and only show compatible packages for older versions if you selected that specific CMS version in the filter.
